app/src/components/dialog/ErrorDialog.tsx

```diff
@@ -100,13 +100,17 @@ export const ErrorDialog: React.FC<IErrorDialogProps> = (props) => {
         {props.dialogError && (
           <DialogContent>
             <DialogContentText id="alert-dialog-description">{props.dialogError}</DialogContentText>
-            <Button color="primary" onClick={() => setIsExpanded(!isExpanded)}>
-              {(isExpanded && 'Hide detailed error message') || 'Show detailed error message'}
-            </Button>
-            {props?.dialogErrorDetails?.length && (
-              <Collapse in={isExpanded}>
-                <ErrorDetailsList errors={props.dialogErrorDetails} />
-              </Collapse>
+            {props?.dialogErrorDetails?.length ? (
+              <>
+                <Button color="primary" onClick={() => setIsExpanded(!isExpanded)}>
```

Comment:
this is causing the errordialog.test.tsx test case to no longer be super accurate
can we change the message on the test to say:
```
renders correctly with a non-detailed error message
```

and then add a test case for with detailed error message :)